app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -53,3 +53,14 @@
     text: "An adviser with years of teaching experience can help you change careers to become a teacher. Chat by phone, text, or email as little or often as you need.",
```

Comment:
Need to remove old component

![image](https://github.com/user-attachments/assets/5d04c31d-a3ed-444a-92ba-a9eeb6bcab15)